linux_os/guide/system/accounts/accounts-session/user_umask/accounts_umask_interactive_users/rule.yml

```diff
@@ -1,6 +1,6 @@
 documentation_complete: true
 
-prodtype: ol7,ol8,rhel7,rhel8,rhv4,wrlinux1019
+prodtype: ol7,ol8,rhel7,rhel8,rhv4,sles15,wrlinux1019
```

Comment:
s/sles/sle/g

Reply:
Forgot to push after I fixed it last night.  I apologize